spec/mailers/user_mailer_spec.rb

```diff
@@ -32,4 +32,24 @@
       @mail.body.encoded.should match("http://localhost:3000/g/#{@group.key}")
     end
   end
+
+  context '#added_to_group' do 
+    before do
+      @user    = double(:user, name: 'Wayne G', email: 'w_recipient@g.com', name_and_email: nil)  
```

Comment:
Rob and I are both moving away from using doubles in the test suite; we're willing to sacrifice a tiny bit of speed for the knowledge that we're actually testing how the application works, instead of the construct we've imagined.

Also, the `.should` syntax, while rampant in our test suite, is less than ideal ([And rspec themselves agree](https://github.com/rspec/rspec-expectations/blob/master/Should.md)). Please use `expect(...).to eq` instead. :+1: